frontend/src/features/table/ContextMenuItem.tsx

```diff
@@ -18,7 +18,9 @@ const ContextMenuItem = ({ title, subtitle, ...props }) => {
           )}
         </Stack>
       }
-      onHide={() => {}}
+      onHide={() => {
+        /* override default onHide */
+      }} // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
```suggestion
      onHide={() => { return }}
```
In the online eslint playground this seems to silence the linter.

Reply:
Looks more elegant, thanks for the suggestion! I would maintain the previous override comment though.

Reply:
Ah, sorry I erroneously removed it.